backend/src/v5/models/metadata.js

```diff
@@ -16,15 +16,14 @@
  */
 
 const db = require('../handler/db');
-const { UUIDLookUpTable } = require('../utils/helper/uuids');
 const { templates } = require('../utils/responseCodes');
 
 const Metadata = {};
 
 const collectionName = (model) => `${model}.scene`;
 
-Metadata.getMetadataById = async (teamspace, model, metadataId, projection) => {
-	const metadata = await db.findOne(teamspace, collectionName(model), { _id: metadataId }, projection);
+Metadata.getMetadataByQuery = async (teamspace, model, query, projection) => {
```

Comment:
I think you should keep getMetadataById? from what I can tell, as it stands that's the only thing you do when you call this function ?